graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -44,6 +51,138 @@ def _are_filter_fields_uniquely_indexed(filter_fields, unique_indexes):
     return False
 
 
+def _try_obtain_parameter_value(argument_name, parameters):
+    """Return value of argument in parameters dict or raise ValueError."""
+    parameter_name = get_parameter_name(argument_name)
+    if parameter_name not in parameters:
```

Comment:
Could we just attempt to access `parameters[parameter_name]` and let Python raise a `KeyError` if the parameter name isn't found? Would eliminate the need for this function

Reply:
Good point.